content/en/docs/Features/ambient.md

```diff
@@ -39,6 +41,41 @@ When a workload, application, or service is part of the Ambient Mesh, a badge wi
 
 ![Workloads Captured by Ambient](/images/documentation/features/ambient/pod-captured.png)
 
+* It is possible to check each pod protocol in the information tooltip. In Ambient, instead of TCP, they use HBONE. 
+
+![Pod details protocol](/images/documentation/features/ambient/protocol.png)
+
+* When a workload traffic is handled by a Waypoint, the workload details will show a link to the proxy:
+
+![Waypoint link](/images/documentation/features/ambient/waypoint-link.png)
+
+### Waypoint proxy details
+
+The workload details for a Waypoint has specific Waypoint data. It is identified with the L7 label: 
+
+![Waypoint label](/images/documentation/features/ambient/waypoint-label.png)
+
+The proxy status shows a new info message when some of the Discovery Services is IGNORED, and there are no other errors: 
+
+![Waypoint proxy status](/images/documentation/features/ambient/waypoint-proxy-status.png)
+
+This condition is usually expected, but it is shown as an info in case it is not. 
+
+The waypoint proxy generates traces for the services that redirects the traffic from, and this is where it can be checked, because the proxy generates the traces with the waypoint service name:
```

Comment:
```suggestion
The waypoint proxy generates traces for the services for which it handles traffic, and this is where it can be checked, because the proxy generates the traces with the waypoint service name:
```